hack/make/.build-rpm/docker-engine.spec

```diff
@@ -144,6 +144,11 @@ install -p -m 644 contrib/udev/80-docker.rules $RPM_BUILD_ROOT/%{_sysconfdir}/ud
 install -d $RPM_BUILD_ROOT/etc/sysconfig
 install -d $RPM_BUILD_ROOT/%{_initddir}
 
+# TODO(runcom): setup this for deb also? right?
```

Comment:
https://github.com/docker/docker/blob/93e8aff1bc4a91e0cfdfd0974b4e68d43cc78006/hack/make/.build-deb/docker-engine.install is the file you're looking for :+1: